codec.go

```diff
@@ -23,6 +23,14 @@ var ErrUnsupportedSampleCodec = errors.New("unsupported sample codec")
 // in this interface.
 var AnySampleCodec = sample.Codec(-1)
 
+// IoReadSeekCloser just wraps io.ReadSeeker and io.Closer, as a convenience
+// for specifying a decoding function which can also seek (codec functions
+// always have a Close())
+type IoReadSeekerCloser interface {
```

Comment:
Rename to ReadSeekCloser; matching [io.ReadWriteCloser](https://godoc.org/io#ReadWriteCloser)